Description: Upgrade Doxyfile to latest doxygen version
 Doxyfile provided by upstream was generated with 1.2.15 and is
 not working with 1.8 version of doxygen shipped by Debian. This
 patch upgrades the Doxyfile which is created using doxygen -u
 command.
 .
 Additionally Doxyfile was referring to non existing templates
 under doc folder which is now removed.
 .
Author: Vasudev Kamath <kamathvasudev@gmail.com>
Forwarded: no
Last-Update: 2013-01-20

--- a/Doxyfile
+++ b/Doxyfile
@@ -91,8 +91,8 @@ IGNORE_PREFIX          =
 GENERATE_HTML          = YES
 HTML_OUTPUT            = html
 HTML_FILE_EXTENSION    = .html
-HTML_HEADER            = ./doc/doxytmpl/header.html
-HTML_FOOTER            = ./doc/doxytmpl/footer.html
+HTML_HEADER            =
+HTML_FOOTER            =
 HTML_STYLESHEET        =
 HTML_ALIGN_MEMBERS     = YES
 GENERATE_HTMLHELP      = NO
